make_level.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import os
+
+path = os.path
+join = os.path.join
+cwd = os.getcwd()
+
+def writeInit(skill_name, skill_path, level_name):
```

Comment:
call this `write_init`

Reply:
Okay, this change will be reflected when I push